webview/platforms/cef.py

```diff
@@ -99,13 +103,18 @@ def initialize(self):
 
         self.browser.ExecuteJavascript(dom.src)
 
-        sleep(0.1) # wait for window.pywebview to load
+        sleep(0.1)  # wait for window.pywebview to load
         self.initialized = True
         self.loaded.set()
 
     def close(self):
         self.browser.CloseBrowser(True)
 
+    def resize(self, width, height):
+        win32gui.SetWindowPos(self.inner_hwnd, win32con.NULL, 0, 0, width - 16, height - 38,
+                              win32con.SWP_NOZORDER | win32con.SWP_NOMOVE | win32con.SWP_NOACTIVATE)
```

Comment:
`windll.user32.SetWindowPos`

Reply:
It's strange but using ctypes.windll I got hangs when I randomly resize window by dragging it (but with much less chance than using cef.WindowUtils.OnSize). In the same case I got no hangs using win32gui during much longer test. But it still could be random....